includes/admin/reporting/reports.php

```diff
@@ -2966,3 +2966,22 @@ function edd_add_screen_options_nonces() {
 	wp_nonce_field( 'meta-box-order',  'meta-box-order-nonce', false );
 }
 add_action( 'admin_footer', 'edd_add_screen_options_nonces' );
+
+/**
+ * This function adds a notice to the bottom of the Tax reports screen if a default tax rate is detected, stating
+ * that we cannot report on the default tax rate.
+ *
+ * @since 3.0
+ * @param $report
```

Comment:
```suggestion

 * @param \EDD\Reports\Data\Report|\WP_Error $report The current report object,
 *                                                   or WP_Error if invalid.
```

Sorry meant to include this in the previous suggestion.